packages/@aws-cdk/aws-logs/lib/log-group.ts

```diff
@@ -273,6 +274,13 @@ export enum RetentionDays {
  * Properties for a LogGroup
  */
 export interface LogGroupProps {
+  /**
+   * The KMS Key to encrypt the log group with.
+   *
+   * @default No KMS key
```

Comment:
does this use the default master key if unspecified?
the default text should describe the behaviour - feel free to adjust if i'm reading it incorrectly.

```suggestion
   * @default - log group is encrypted with the default master key
```